### PR TITLE
WIP 771 fix failed pipelines

### LINKED ISSUE #771: Fix failed pipeline
https://github.com/pawelru/ie_status?tab=readme-ov-file

### PR DESCRIPTION
**Pull request by m7pr**
Fixes #771 
- revdepcheck 🆗  https://github.com/insightsengineering/teal.modules.general/actions/runs/11162906687
- rhub 🌮 WIP https://github.com/insightsengineering/teal.modules.general/actions/runs/11211837075/job/31161572229

This fixes R-hub and revdepcheck checks

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -81,7 +81,7 @@ Suggests:
 VignetteBuilder:
     knitr
 Config/Needs/verdepcheck: haleyjeppson/ggmosaic, tidyverse/ggplot2,
-    rstudio/shiny, insightsengineering/teal,
+    rstudio/shiny, insightsengineering/teal, insightsengineering/teal.slice,
```
Comment (m7pr) [inline DESCRIPTION:84]: unsure if teal.slice is needed here for verdepcheck

File: vignettes/teal-modules-general.Rmd [hunk 2]
```diff
@@ -146,7 +146,7 @@ In a `teal` app, data and modules are decoupled. In the app above:
 - The app developer specified the module and assigned it to the `modules` argument.
 - The `init` function took these arguments and returned a list containing `ui` and `server` object, which can be demonstrated by running:
 
-```{r, indent = "     "}
+```{r, indent = "     ", eval = requireNamespace("sparkline")}
```
Comment (m7pr) [inline vignettes/teal-modules-general.Rmd:149]:
```suggestion
```{r, indent = "     ", eval = requireNamespace("sparkline", quietly = TRUE))}
```

File: vignettes/teal-modules-general.Rmd [hunk 3]
```diff
@@ -39,7 +39,7 @@ See [package functions / modules](https://insightsengineering.github.io/teal.mod
 ### Example application
 
 A simple application featuring the `tm_variable_browser()` module:
-```{r app, message = FALSE, results = "hide"}
+```{r app, message = FALSE, results = "hide", eval = requireNamespace("sparkline")}
```
Comment (m7pr) [inline vignettes/teal-modules-general.Rmd:42]:
```suggestion
```{r app, message = FALSE, results = "hide", eval = requireNamespace("sparkline", quietly = TRUE))}
```

File: DESCRIPTION (hunk 1 above)
Comment (m7pr) [inline DESCRIPTION:84]:
```suggestion
    rstudio/shiny, insightsengineering/teal, 
```

**Comment by m7pr** [comment]
Hey @donyunardi, as agreed, I fixed `r-hub` checks and `revdepcheck`. This package is still failing in Scheduled job becuase of the failures of dependencies-test job (`verdepcheck`). I created a separate issue to handle that
https://github.com/insightsengineering/teal.modules.general/issues/790

**Review by donyunardi — APPROVED** [review]

**Pull request merged**